CMakeLists.txt

```diff
@@ -186,7 +186,8 @@ add_test(NAME "libfacter\\ tests" COMMAND libfacter_test)
 if (RUBY_FOUND)
     find_program(BUNDLER_PATH NAMES bundle.bat bundle)
     if (BUNDLER_PATH)
-        execute_process(COMMAND ${BUNDLER_PATH} --retry 2 --gemfile=lib/Gemfile --quiet)
+        message(STATUS "Bundler found, installing dependencies for Ruby tests...")
+        execute_process(COMMAND ${BUNDLER_PATH} --retry 2 WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/lib")
```

Comment:
Does this need to be `COMMAND ${BUNDLER_PATH} install`?

Reply:
AFAIK, running bundler with no argument should default to bundle install, but it doesn't hurt to be more verbose...

Reply:
updated on my PR: https://github.com/puppetlabs/facter/pull/1880